spec/service_actor/result_spec.rb

```diff
@@ -14,7 +14,7 @@
         result = described_class.new
         result.name = ""
 
-        expect(result.name?).to eq false
+        expect(result.name?).to be false
```

Comment:
These changes, which were made by the newer version of `rubocop-rspec`, can stay.  No harm...